x-pack/plugins/endpoint/public/applications/endpoint/components/header_nav.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { MouseEvent } from 'react';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import { EuiTabs, EuiTab } from '@elastic/eui';
+import { useHistory, useLocation } from 'react-router-dom';
+
+export interface NavTabs {
+  name: string;
+  id: string;
+  href: string;
+}
+
+export const navTabs: NavTabs[] = [
+  {
+    id: 'home',
+    name: i18n.translate('xpack.endpoint.headerNav.home', {
+      defaultMessage: 'Home',
+    }),
+    href: '/',
+  },
+  {
+    id: 'management',
+    name: i18n.translate('xpack.endpoint.headerNav.management', {
+      defaultMessage: 'Management',
+    }),
+    href: '/management',
+  },
+  {
+    id: 'alerts',
+    name: i18n.translate('xpack.endpoint.headerNav.alerts', {
+      defaultMessage: 'Alerts',
+    }),
+    href: '/alerts',
+  },
+  {
+    id: 'policies',
+    name: i18n.translate('xpack.endpoint.headerNav.policies', {
+      defaultMessage: 'Policies',
+    }),
+    href: '/policy',
+  },
+];
+
+const Tabs = styled(EuiTabs)`
```

Comment:
❔ There is also `EuiTabbedContent` which seems like it would capture some more of what you're trying to accomplish here. Can you elaborate on why you didn't use the `EuiTabbedContent` approach?

Reply:
@bkimmel that styled doesn't make a difference that I can see, so I removed it. m See this for the original context https://github.com/elastic/kibana/pull/58604#discussion_r384750010